BBB/Source/Item/BeerSearch/BeerSearchVC.swift

```diff
@@ -0,0 +1,92 @@
+
+import Foundation
+import UIKit
+import Then
+import SnapKit
+import Alamofire
+import Kingfisher
+
+final class BeerSearchVC: BaseVC, UISearchBarDelegate{
+    
+    private let searchBar = UISearchBar().then{
+        $0.placeholder = "Search"
+        $0.searchBarStyle = .minimal
+    }
+    private let beerImageView = UIImageView()
+    
+    private let idLabel = UILabel().then{
+        $0.textColor = .black
+        $0.font = UIFont(name: "Helvetica", size: 12)
+    }
+    private let descriptionLabel = UILabel().then{
+        $0.font = UIFont(name: "Helvetica-bold", size: 16)
+        $0.textAlignment = .center
+    }
+    let urlString = "https://api.punkapi.com/v2/beers/"
```

Comment:
what is that?

Reply:
7067a1a19c66bc58d95de2c780ca23621e8c50cf